osx32/build

```diff
@@ -1,13 +1,20 @@
 #!/bin/bash
 
 CompilerOptions="-DDFRE_OSX32 "
-CompilerOptions+="-nostdlib "
 CompilerOptions+="-std=c++11 "
 CompilerOptions+="-fno-exceptions -fno-asynchronous-unwind-tables "
 CompilerOptions+="-fno-stack-protector "
 CompilerOptions+="-m32 "
 CompilerOptions+="-Wall -Werror -Wextra "
 CompilerOptions+="-Wno-unused-parameter "
 
+if [ "$1" == '--coverage' ]; then
+    echo "To get a coverage report run: ./test_re && gcov tester.cpp"
+    CompilerOptions+="-fprofile-arcs -ftest-coverage -DNO_START "
+else
+    CompilerOptions+="-nostdlib "
+    CompilerOptions+="-mmacosx-version-min=10.6 "
```

Comment:
I think the macosx version flag could go outside the if statement. Unless profiling doesn't work when you use it.

Or is that version flag needed to get nostdlib to work?

Reply:
.

Reply:
Pretty sure it was needed, but considering you should be able to target OSX 10.6 anyway I think just including it with the 'normal' options is best. Addressed in 09a0941